### DISCUSSION TITLE
Keyball61 setup

### DISCUSSION THREAD

**Discussion by therealandydyck** [Yowkees/keyball#683] [Q&A] [answered]
Hello,

I've been scouring everywhere I can for a solution to my issue but I have had no luck so far.

When I connect my SeaPicro to my Win10 computer, it is not recoginzed as a HID device and opens up a folder.
I have read that this is likely the bootloader mode and a .uf2 file can be dragged into the folder to flash the firmware.
The folder is supposed to disappear when dropping in a uf2 file, but this does not happen, it will stay there until the reset button is pressed or the controller is unplugged.

When quickly pressing the reset button the device reconnects and the folder pops up again like nothing happened.
When disconnecting and reconnecting the device, the folder pops up again like its in bootloader mode.

To my understanding the SeaPicro should not be defaulting to bootloader mode and I cannot find a solution for this.
Any help would be greatly appreciated.

**Comment by Yowkees** [comment]
I have no idea about Sea-Picro.
Please someone who is good at RP2040 help us!

**Reply by therealandydyck** [reply, marked as answer]
So there's an issue with the instructions to install via remap for the sea-picro
It was not picked up via the automated flash function
Using the 'copy' function let me save a .uf2 file, but it was just named firmware.uf2 and the sea-picro did not do anything with it.

I discovered the pre-compiled and named files below buried in the generic 'Firmware' section of the holykeebs documentation. 
When I copied it over it behaved like expected.

https://github.com/idank/qmk_firmware/releases/tag/holykeebs-master-latest 

This should probably be put in the keyboard specific section of the instructions instead of pointing people toward the QMK compiler and generic documentation
It would probably be a good idea to realign the naming scheme to match the keyboards and brands, as idank and sweeq and spankbd are non-obvious and I would have never found them via browsing the QMK keyboards directory or otherwise

**Comment by jtuyen** [comment]
Try and restore the seapicro firmware: https://github.com/joshajohnson/sea-picro/releases/tag/v1.0 and then upload your uf2 file.

**Reply by therealandydyck** [reply]
Could you link to/provide some instructions on how to restore the firmware?  If I were well versed enough in this that I could take that link and perform the non-obvious steps I probably wouldn't be posting a question here lol.

Thank you,

**Comment by lorenasilvaaraujo** [comment]
Hi, did you manage to setup your keyboard. I just got mine assembled from Holykeebs but I might have the same problem as you, it does not work. Also noobie here in keyboard firmware.
